### ISSUE TITLE
H61A0 flashes a bright every so often

### ISSUE THREAD

**Issue by petebutty** [wez/govee2mqtt#45]
### Govee Device SKU

H61A0

### Govee2MQTT Version

17.0.2

### Describe the issue

H61A0 lights are recognized and can be controlled fine, but every minute or so it blinks a bright flash for a fraction of a second, 

### Startup Diagnostics

Error shown is like this...
[2024-01-10T19:23:13 ERROR govee::service::iot] Decoding IoT Packet: govee::service::iot::start_iot_client::{{closure}}::{{closure}}::Packet op.command[0]: packet must contain 20 bytes, have 4 at line 1 column 274. Input: {"proType":2,"sku":"H61A0","device":"56:DC:D4:AD:FC:1C:6A:92","softVersion":"2.07.01","wifiSoftVersion":"2.05.08","wifiHardVersion":"1.02.00","cmd":"ptIotOp","type":2,"transaction":"u_1704914593781","pactType":2,"pactCode":1,"state":{"result":1},"op":{"command":["6gEB6g=="]}} {"proType":2,"sku":"H61A0","device":"56:DC:D4:AD:FC:1C:6A:92","softVersion":"2.07.01","wifiSoftVersion":"2.05.08","wifiHardVersion":"1.02.00","cmd":"ptIotOp","type":2,"transaction":"u_1704914593781","pactType":2,"pactCode":1,"state":{"result":1},"op":{"command":["6gEB6g=="]}}

Not entirely sure what this all means though?

**Comment by wez** [comment]
I have a couple of govee devices that do this intermittently, and have done so for months before I even thought of building this addon: I think they're just bad and/or failing. The quality of the hardware isn't the best.  I've had one floor lamp completely burn out its controller, my glide lights will occasionally start to flicker intermittently, and I have another floor lamp whose top segment sometimes flickers.  That's 3 out of 11 devices with some kind of hardware quality issue.

**Comment by wez** [comment]
The error in the logs you shared is harmless and unrelated to the symptoms you described, but I'll do something to account for it and make it go away in a later release.

**Comment by petebutty** [comment]
Really? Sorry to hear that, all my devices have been working great since I
bought them, which was well over a year ago, could be the law of averages
though, and I finally got one device that's gonna shit the bed.
I was using govee lan control before and had never had an issue, I'll maybe
need to check the strip itself then, thanks

**Comment by wez** [comment]
I'm going to close this because it isn't related to this software.

**Comment by petebutty** [comment]
That's cool, I've actually not had an issue since, thanks for my life
easier though, all the best for 2024 wez

**Comment by freakadings** [comment]
Hey ho,
I got the same problem with my Glide Hexa Panels (H6061). When turned off, they turn on for ~100ms. I removed and added the panels again to the govee app, which worked for someone and for me for one evening. On the next day it was flickering again. 
As @wez mentioned it's probably not an issue with the govee2mqtt addon but with the "old" "Govee LAN Control" integration it never happened. So i rolled back and the flickering dissappeared again but I want to use the effects of my lamps within Home Assistant and is far as i found out it's not possible with the lan integration :/
